thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/MergedAnomalyResultManager.java

```diff
@@ -42,5 +42,7 @@ MergedAnomalyResultDTO findLatestConflictByFunctionIdDimensions(Long functionId,
 
   List<MergedAnomalyResultDTO> findByTime(long startTime, long endTime);
 
+  List<MergedAnomalyResultDTO> findUnNotifiedByFunctionIdAndIdLesserThanAndEndTimeGreaterThanLastOneDay(long functionId, long anomalyId);
```

Comment:
It's hard to tell what this function is for. I assume it's related to sending out email summaries. Is there a better way of naming it?